packages/common-ui-web/src/components/VehicleDynamicWireframe/VehicleDynamicWireframe.tsx

```diff
@@ -0,0 +1,45 @@
+import { useMonkTheme } from '@monkvision/common';
+import { wireFrame } from '@monkvision/sights';
+import { PartSelectionOrientation, VehicleModel, VehiclePart } from '@monkvision/types';
+import { useMemo } from 'react';
+import { DynamicSVG } from '../DynamicSVG';
+import { usePartsOverlay } from './hooks';
+
+export interface VehicleDynamicWireframeProps {
+  vehicleModel: VehicleModel;
+  orientation?: PartSelectionOrientation;
+  parts: VehiclePart[];
+  /**
+   * emit when change in the selected parts.
+   */
+  onPartsSelected?: (parts: VehiclePart[]) => void;
```

Comment:
I changed my mind and I think we should change this prop's name to `onClickPart`